README.md

```diff
@@ -32,17 +32,49 @@ lit tests/filecheck
 
 ## Generating executables through MLIR
 
-xDSL can generate executables using MLIR as the backend. To use this
-functionality, make sure to have clang, mlir-opt and mlir-translate installed
-and in the PATH. Clang can be installed by standard package managers (apt,
-pacman...), for MLIR follow [this](https://mlir.llvm.org/getting_started/) and
-run `ninja install` afterward. Given an input file `input.xdsl`, that contains
-IR with only the mirrored dialects found in `src/xdsl/dialects` (arith, builtin,
-cf, func, irdl, llvm, memref, and scf), run:
+xDSL can generate executables using MLIR as its backend.
+To benefit from this functionality, we first need to clone and build MLIR: https://mlir.llvm.org/getting_started/
+
+```bash
```

Comment:
Do we want this in our README? Isn't it enough to point to the `Getting Started` website of MLIR?